api/controllers/CommentController.js

```diff
@@ -20,40 +20,60 @@ const presentComment = (comment) =>
     return Object.assign(buckets, c)
   })
 
-const createAndPresentComment = function (commenterId, text, post, opts = {}) {
+const createAndPresentComment = function (commenterId, text, post, parentComment, opts = {}) {
```

Comment:
general design guideline: if you go past two or so args, start putting them into an options object. reading the argument names is much easier to follow than remembering arguments by their order. so in this case i would move `parentComment` into `opts`